src/components/userLogin_comp.js

```diff
@@ -0,0 +1,33 @@
+import { OnboardingHeader } from './onboardingHeader'
+
+
+function UserLoginComp() {
+return (
+    <form>
+        <OnboardingHeader h1={'Welcome back'} p ={'Enjoy extra features when you create an account with us.'}/>
+
+        <div className='flex justify-between items-center my-6'>

```

Comment:
To ensure consistency. Remove the `my-6` and just use a `space-y-6` in the form node.

Remove every other part you find it